recipe/meta.yaml

```diff
@@ -24,22 +24,21 @@ requirements:
     - einops >=0.3,<0.7
     - fiona >=1.8,<2
     - kornia >=0.6.5,<0.6.10
+    - lightning >=1.8,<2
     - matplotlib-base >=3.3,<4
     - numpy >=1.17.2,<2
-    - omegaconf >=2.1,<3
     - pillow >=6.2,<10
     - pyproj >=2.2,<4
     - python >=3.7,<4
-    - pytorch >=1.12,<2
-    - pytorch-lightning >=1.5.1,<2
+    - pytorch >=1.12,<3
     - rasterio >=1.0.20,<2
     - rtree >=1,<2
     - scikit-learn >=0.21,<2
     - segmentation-models-pytorch >=0.2,<0.4
     - shapely >=1.3,<3
     - timm >=0.4.12,<0.7
     - torchmetrics >=0.10,<0.12
-    - torchvision >=0.13,<0.15
+    - torchvision >=0.13,<0.16
 
     # https://github.com/conda-forge/pytorch-lightning-feedstock/issues/113
     - lightning-utilities ==0.3.*
```

Comment:
Try removing this line, it was specific to pytorch-lightning

Reply:
No dice unfortunately

Reply:
You still need to increase the upper bound on lightning. From Slack:

> I guess we should just make it `>=1.8,<3` and remove `pip check` until the 0.4.2 release

Reply:
Yes, the new `lightning` conda package you are using only has `2.0.0` and greater on conda-forge